spring-cloud-stream-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -76,104 +77,132 @@ public class StreamApplicationTests {
 }
 ----
 
-== Spring Cloud Stream Main Concepts
+== Main Concepts
 
-Spring Cloud Stream provides a number of abstractions and primitives that simplify writing message-driven microservices.
-In this section we will provide an overview of:
+Spring Cloud Stream provides a number of abstractions and primitives that simplify the writing of message-driven microservice applications.
+This section gives an overview of the following:
 
-* Spring Cloud Stream application model together with the Binder abstraction
-* Persistent publish-subscribe and consumer group support
-* Partitioning
-* Pluggable Binder API
+* A standard application model
+* The Binder abstraction
+* Persistent publish-subscribe support
+* Consumer group support
+* Partitioning support
+* A pluggable Binder API
 
 
-=== Application structure
+=== Application Model
 
-A Spring Cloud Stream application consists of a middleware-neutral core that communicates with the outside world through input and output channels.
-The channels are managed and injected into it by the framework, and a `Binder` connects them to the external brokers.
-Different `Binder` implementations exist for different types of middleware, such as https://github.com/spring-cloud/spring-cloud-stream/tree/master/spring-cloud-stream-binders/spring-cloud-stream-binder-kafka[Kafka], https://github.com/spring-cloud/spring-cloud-stream/tree/master/spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit[Rabbit MQ], https://github.com/spring-cloud/spring-cloud-stream-binder-redis[Redis] or https://github.com/spring-cloud/spring-cloud-stream-binder-gemfire[Gemfire], and an extensible API allows you to write your own `Binder`.  There is also https://github.com/spring-cloud/spring-cloud-stream/blob/master/spring-cloud-stream-test-support/src/main/java/org/springframework/cloud/stream/test/binder/TestSupportBinder.java[TestSupportBinder] that leaves the channel as-is so a test author can interact with the channels directly and easily assert on what is received.
+A Spring Cloud Stream application consists of a middleware-neutral core.
+The application communicates with the outside world through input and output _channels_ injected into it by Spring Cloud Stream.
+Channels are connected to external brokers through middleware-specific Binder implementations.
 
 .Spring Cloud Stream Application
-image::SCSt-with-binder.png[width=300,scaledwidth="50%"]
+image::images/SCSt-with-binder.png[width=300,scaledwidth="50%"]
```

Comment:
This will unfortunately throw off the maven build as the build process inserts the `images` path.